hack/devtools/local_dev_env.sh

```diff
@@ -189,10 +189,8 @@ create_platform_identity_and_assign_role() {
     echo "Name: $name"
     echo ""
 
-    # The following operators require a role assignment since they need access to customer BYO virtual network
-    # RP will be responsible for other role assignments
-    if [[ "${operatorName}" == "MachineApiOperator" || "${operatorName}" == "NetworkOperator" \
-        || "${operatorName}" == "AzureFilesStorageOperator" || "${operatorName}" == "ServiceOperator" ]]; then
+    # Storage Operator don't require access to customer BYO virtual network
+    if [[ "${operatorName}" != "StorageOperator" ]]; then
```

Comment:
What about image registry operator? or cloud controller manager?

Reply:
@cadenmarchese mentioned except `StorageOperator` all other operators would need access to Vnet as per https://issues.redhat.com/browse/OCPBUGS-28762

Reply:
For the GA, since CLI isn't creating the identities, we also assume that the customer running the create cluster won't be having the permission to assign RBAC over the vnet.
Thus it is being assumed that the provided identities to the CLI during cluster create/update will have the RBAC already performed.